checks/http/dnsme.rb

```diff
@@ -0,0 +1,29 @@
+module Intrigue
+module Ident
+module Check
+class DNSME < Intrigue::Ident::Check::Base
```

Comment:
Class names should camelcase http://rubylearning.com/satishtalim/ruby_names.html

Reply:
In this case, "Dnsme" is expected